colossalai/booster/plugin/gemini_plugin.py

```diff
@@ -391,11 +440,32 @@ def configure(
             # model = nn.SyncBatchNorm.convert_sync_batchnorm(model, None)
 
             # wrap the model with Gemini
-            model = GeminiDDP(model, **self.gemini_config, verbose=self.verbose)
+            self.dp_group = None
+            self.tp_group = None
+            try:
+                dp_size = dist.get_world_size() // self.tp_size
```

Comment:
Distributed info should be initialized in `__init__()`

Reply:
updated